modules/apps/site/site-insurance-site-initializer/src/main/zippableResources/fragments/raylife/fragments/selected-quote/index.html

```diff
@@ -0,0 +1,14 @@
+<div class="container quote-comparison">
+	
+	<h1 data-lfr-editable-id="quote-comparison-title-selected" data-lfr-editable-type="text">
+		Awesome choice!
+	</h1>
+	<h2>
+		You’re almost finished.
+	</h2>
+	<lfr-widget-web-content>
+	</lfr-widget-web-content>
+	<div class="quote-application">
```

Comment:
Modificar o nome da class para um nome indique a qual escopo ela está sendo aplicada.

Poderia ser algo do tipo:
```suggestion
	<div class="quote-comparison-selected-application">
```
Ajustar também outros pontos que fazem uso desta class.